ypathgen/pathgen.go

```diff
@@ -173,6 +173,11 @@ type GenConfig struct {
 	// If any key is not a wildcard, then this flag doesn't apply, since
 	// all key values must now be specified in the path.
 	SimplifyWildcardPaths bool
+	// SplitByModule controls whether to generate a go package for each yang module.
+	SplitByModule       bool
+	FakeRootPackageName string
+	TrimOCPackage       bool
+	BaseImportPath      string
```

Comment:
I'm assuming you will add comments for these?

Reply:
added